Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonSettings.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.covidcertificate.person.core
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.joda.time.Instant
+
+data class PersonSettings(
+    @JsonProperty("lastSeenBoosterRuleIdentifier")
+    val lastSeenBoosterRuleIdentifier: String? = null,
+
+    @JsonProperty("lastBoosterNotifiedAt")
+    val lastBoosterNotifiedAt: Instant? = null,
+
+    @JsonProperty("showDccReissuanceBadge")
+    val showDccReissuanceBadge: Boolean = false,
```

Comment:
Is it enough to store a `Boolean` here or do we need to store the information about which certificate can be re-issued? Because I think it is similar to the Booster Rules - if there is a new certificate that can be replaced we need to show the badge and in order to know if it is a new certificate, we have to store the old one.

Reply:
I thought the same, but the specs is clear on that if the there is reissuance info in the dcc wallet info and it did not exist before that notification and therefore the badge should be shown, it did not state if they were different as in booster case .

/cc @mlenkeit fyi

Reply:
in any case I think flag is `enough`